### PR TITLE
Support for Injecting Beans Directly In Templates

### LINKED ISSUE #546: Support for Injecting Beans Directly In Templates
See https://quarkus.io/guides/qute-reference#injecting-beans-directly-in-templates

See sample https://github.com/FroMage/quarkus-renarde-todo/blob/5384be6c070d11e5aff200f2469d969c100af088/src/main/resources/templates/tags/input.html#L7

Today we have a warning:

image

### PR DESCRIPTION
**Pull request by angelozerr**
Support for Injecting Beans Directly In Templates

Fixes #546

Signed-off-by: azerr <azerr@redhat.com>

### CODE REVIEW

**Comment by angelozerr** [comment]
Here a demo with inject namespace:

InjectQuteDemo

**Comment by angelozerr** [comment]
@fbricon the PR is ready to review it.

File: qute.jdt/com.redhat.qute.jdt.test/src/main/java/com/redhat/qute/jdt/template/TemplateGetDataModelProjectTest.java [hunk 1]
```diff
@@ -212,24 +215,64 @@ private static void testValueResolvers(DataModelProject<DataModelTemplate<DataMo
...
+				"bar", resolvers);
+
+		// @Named("user")
+		// private String getUser() {...
+		assertValueResolver("inject", "getUser() : java.lang.String", "org.acme.qute.InjectedData", //
+				"user", resolvers);
+
+		// @Named
+		// private String getSystemUser() {...
+		assertValueResolver("inject", "getSystemUser() : java.lang.String", "org.acme.qute.InjectedData", //
```
Comment (fbricon) [inline qute.jdt/com.redhat.qute.jdt.test/src/main/java/com/redhat/qute/jdt/template/TemplateGetDataModelProjectTest.java:254]: I don't think that's a valid use case. CDI would expose a bean named getSystemUser or maybe systemUser (but I'm not that familiar with CDI).
I suggest you try running an actual Quarkus app with this @Named beans to check whether it makes sense or not.

**Comment by fbricon** [comment]
I feel like we show too much crap with:
Screenshot 2022-03-04 at 16 43 01

Useful variables and properties are at the bottom of the list, you need to scroll down. Can we lower the `inject:` items priority?

**Comment by angelozerr** [comment]
> Bunch of NPEs I've seen today:

It should be fixed with test.

**Comment by angelozerr** [comment]
> Useful variables and properties are at the bottom of the list, you need to scroll down. Can we lower the inject: items priority?

It should b efixed

**Pull request merged**